spyder/widgets/variableexplorer/dataframeeditor.py

```diff
@@ -24,6 +24,7 @@
                             QHeaderView)
 
 from pandas import DataFrame, DatetimeIndex, Series
+from pandas._libs.tslib import OutOfBoundsDatetime

```

Comment:
The `pandas._libs` namespace is only available for in Pandas 0.20, but we support much older versions of Pandas.

How could we address that?

Reply:
Oh! I was not aware of that, sorry. What is the oldest version we should support? We can access it from `from pandas.tslib import OutOfBoundsDatetime` in version 0.16. I can check for even older versions. The import will then depend on the pandas version.

Reply:
> The import will then depend on the pandas version.
 
Yep, you can change this to

```python
try:
    from pandas._libs.tslib import OutOfBoundsDatetime
except:
    from pandas.tslib import OutOfBoundsDatetime
```

That will work for Pandas 0.13.1+, which is the oldest version we *supposedly* support.

If you think we should update that to a more recent version, feel free to do it in another PR for 3.x :-)

Reply:
Done! Thanks.
For the moment I do not see any strong reason to update the requirement for pandas version but maybe when working on #6008. Is anyone still using 0.13 somewhere? 😄